recipes/cppcheck/all/conandata.yml

```diff
@@ -1,13 +1,10 @@
 sources:
-  "2.10":
-    url: "https://github.com/danmar/cppcheck/archive/2.10.tar.gz"
-    sha256: "785dcbf711048dfe43ae920b6eff2eeebb4a096e88188a40e173ca4c030f57c3"
+  "2.10.3":
+    url: "https://github.com/danmar/cppcheck/archive/2.10.3.tar.gz"
+    sha256: "8aae5e116daeaaf5d19f3efa61b91c06f161cb97412a1d1af6e1e20686e48967"
```

Comment:
Why do you remove 2.10 ?

Reply:
Isn't it better to keep the latest bugfix version, especially for a tool?

Reply:
Here is the general policy for old version management: https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/sources_and_patches.md#removing-old-versions

Reply:
OK, as I understand, the last 3 minor versions shall be kept, in this case 2.8.2, 2.9.3 and 2.10.3. So, according to the rules you mentioned, I dont see why 2.10 shall be kept?

Reply:
It's fine like that.

Reply:
> from time to time we remove old versions mainly due to technical reasons

:) we don't remove them for no reason, if the build fails or the maintenance cost is a burden too high then its acceptable.

Given the package and recipe still work I would appreciate leaving them :)

Reply:
The recipe doesn't work for 2.7.X because z3 is not conan v2 ready

Reply:
That's fine v2 is not required anyways

Please keep 2.10 and 2.9.2 since there no reason to remove them

Reply:
I reverted the changes